great_expectations/experimental/metric_repository/cloud_data_store.py

```diff
@@ -11,11 +13,41 @@
 
 
 class CloudDataStore(DataStore[StorableTypes]):
+    def __init__(self, context: AbstractDataContext):
+        super().__init__(context=context)
+        config = self._context._cloud_config
+        self._session = create_session(access_token=config.access_token)
+        self._url = (
+            f"{config.base_url}/organizations/{config.organization_id}/metric-runs"
+        )
+
+    def _construct_json_payload(self, metric_run: MetricRun):
+        result = {
+            "id": str(metric_run.id),
+            "metrics": [],
+        }
+        for metric in metric_run.metrics:
+            result["metrics"].append(
+                {
+                    "id": str(metric.id),
+                    "batch_id": metric.batch.id,
+                    "data_asset_id": str(metric.batch.data_asset.id),
+                    "metric_name": metric.metric_name,
```

Comment:
I've been thinking that maybe we can get rid of metric_name since we have metric_type. If we need the name we can associate it with the type on either side of the API but no need to send it in the payload. What do you think?

Reply:
The metric_type of TableMetric is the same for both table.columns and table.row_count. I don't see anything wrong with keeping the metric_name included.

Reply:
Good point and good catch 🙇